tools/utils/utils.js

```diff
@@ -534,9 +534,9 @@ exports.execFileSync = function (file, args, opts) {
     });
 
     return {
-      success: ! new Promise(function (resolve) {
+      success: await !new Promise(function (resolve) {
```

Comment:
`! promise` is always false. This needs to instead use `! (await ...)` so it uses the resolved value.